zuul.d/nodeset.yaml

```diff
@@ -313,6 +313,23 @@
       - name: ocps
         nodes:
           - crc
+# todo: Remove. Temporal. Needed as the credentials used in ci-bootstrap jobs for IBM don't work
```

Comment:
(non-blocking) question: Do we expect to eventually roll back to IBM? I like the idea of the parent change so rolling that back it's just creating another child with the previous network configuration. Sounds like a good change IMHO.

Reply:
Yes we intend to roll back to using a cloud neutral label/nodeset but for the short to mid term it's not an option. The infra team can't provide us the same restricted access we need on IBM cloud as we have for Vexx, once they have time to implement the needed changes we can remove the pin to Vexx.